examples/seismic/model.py

```diff
@@ -168,21 +168,21 @@ def demo_model(preset, **kwargs):
         # By default, the top part of the domain has 1.5 km/s,
         # and the bottom part of the domain has 2.5 km/s.
         shape = kwargs.pop('shape', (101, 101))
-        spacing = kwargs.pop('spacing', tuple([10. for _ in shape]))
+        spacing = kwargs.pop('spacing', tuple([1. for _ in shape]))
```

Comment:
Any reason why?

Reply:
So that if a user uses a pre-set model the results don't look 'crap'. Any reason for it not be 1?